packages/visual-editor/src/components/atoms/enhancedCta.tsx

```diff
@@ -0,0 +1,228 @@
+import * as React from "react";
+import { Link, LinkType } from "@yext/pages-components";
+import { Button, ButtonProps } from "./button.js";
+import { themeManagerCn } from "@yext/visual-editor";
+import { FaAngleRight } from "react-icons/fa";
+import { getDirections } from "@yext/pages-components";
+import { PresetImageType } from "../../types/types";
+
+// Preset image icons mapping
+const presetImageIcons: Record<PresetImageType, React.ReactNode> = {
+  phone: (
```

Comment:
Any reason we're not using the font awesome icons or whatever icon library we're using in VE?